core/lib/utils/string_convert.dart

```diff
@@ -5,7 +5,7 @@ import 'package:core/utils/app_logger.dart';
 import 'package:core/domain/exceptions/string_exception.dart';
 
 class StringConvert {
-  static const String separatorPattern = r'[ ,;]+';
+  static const String emailSeparatorPattern = r'[,;]+';
```

Comment:
try to support as current version of customer: keep ` ` (space)

Reply:
Done